internal/controller/persistentvolume/persistentvolume.go

```diff
@@ -161,11 +179,18 @@ func (r ReconcilePersistentVolume) reconcilePV(obj runtime.Object) error {
 		}
 		defer cr.DeleteCredentials()
 
-		err = rbd.RegenerateJournal(imageName, volumeHandler, pool, journalPool, requestName, cr)
```

Comment:
@Madhu-1 we call `regeneratejournal` without any locking in place, so while reconciling a PV, if regenrateJournal() was called more than once, Isnt there a possibility of a race scenario ?